plugins/modules/swim_workflow_manager.py

```diff
@@ -620,6 +703,46 @@ def get_image_id(self, name):
 
         return image_id
 
+    def get_cco_image_id(self, cco_image_name):
+        """
+    Retrieve the unique image ID from Cisco.com based on the provided image name.
+
+    Parameters:
+        self (object): An instance of a class used for interacting with Cisco DNA Center.
+        cco_image_name (str): The name of the software image to search for on Cisco.com.
+
+    Returns:
+        str: The image ID corresponding to the given image name.
+
+    Raises:
+        AnsibleFailJson: If the image ID cannot be found in the response.
+
+    Description:
+        This function sends a request to Cisco DNA Center to retrieve a list of software images
```

Comment:
Use Catalsyt Center instead of DNA Center.

Reply:
updated the code